### PR TITLE
Add image fields partial in the design system

### PR DESCRIPTION
**Pull request by davidgisbey**
## Description

This adds in the image two partials, one for Case Studies and another for all other Edition types.

Currently, it works for uploading a single image, but still needs to work to allow multiple file uploads at the same time.

### Case studies

image

### Other editions 

image

## Trello card

https://trello.com/c/YbYQICAe/684-move-edit-edition-page-to-the-govuk-design-system

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/views/admin/editions/_image_template.html.erb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <%= hidden_field_tag "edition[images_attributes][#{index}][_destroy]", "1" %>
+
+        <%= render "govuk_publishing_components/components/checkboxes", {
+          name: "edition[images_attributes][#{index}][_destroy]",
+          id: "edition_images_#{index}__destroy",
+          items: [
+            {
+              label: "(uncheck to remove)",
+              value: "0",
+              checked: true,
```
Comment (ollietreend) [inline app/views/admin/editions/_image_template.html.erb:55]: With the hardcoded `checked: true` option, I think this means the checkboxes will become re-checked if there are other validation errors on the page.

I think the user's expectation would be that the state of these checkboxes remains intact.

**Review by ollietreend — COMMENTED** [review]
Nice work 👍🏻 this is looking good.

Just one comment from me.

File: app/views/admin/editions/_image_template.html.erb (hunk 1 above)
Comment (davidgisbey) [reply app/views/admin/editions/_image_template.html.erb]: Good catch. Have fixed 👍 thanks for the review

**Review by ollietreend — APPROVED** [review]
Nice 👍🏻 looks good

**Pull request merged**